src/PKSim.Core/CoreConstants.cs

```diff
@@ -201,7 +201,12 @@ public static class Alias
          public static readonly string NON_COMPETIVE_INHIBITION_I = "In";
          public static readonly string MIXED_COMPETIVE_INHIBITION_I = "Im";
          public static readonly string IRREVERSIBLE_INHIBITION_I = "IcTD";
-         public static readonly string K_WATER = "K_water";
+         public static readonly string COMPETIVE_INHIBITION_K_WATER = "K_water_c";
+         public static readonly string UNCOMPETIVE_INHIBITION_K_WATER = "K_water_u";
+         public static readonly string NON_COMPETIVE_INHIBITION_K_WATER = "K_water_n";
+         public static readonly string MIXED_COMPETITIVE_INHIBITION_K_WATER = "K_water_mc";
+         public static readonly string MIXED_UNCOMPETITIVE_INHIBITION_K_WATER = "K_water_mu";
```

Comment:
`MIXED_UNCOMPETITIVE_INHIBITION_K_WATER` is not used (we need only one K_water for mixed inhibition, so I would delete this one and change
`MIXED_COMPETITIVE_INHIBITION_K_WATER = "K_water_mc"` to
`MIXED_INHIBITION_K_WATER = "K_water_m"`

Agree?

Reply:
![grafik](https://user-images.githubusercontent.com/25061876/86472695-de656e00-bd3f-11ea-91c6-d7d6630fcd23.png)
Have forgotten all of that...

Reply:
Actually, `MIXED_COMPETIVE_INHIBITION_I `could be renamed to `MIXED_INHIBITION_I `as well (only the constant, the Alias is ok)